tutorials/continuous-aggs-tutorial.md

```diff
@@ -0,0 +1,238 @@
+time_bucket expression and the time_bucket expression uses the time dimension 
+column of the *rides* hypertable.
+```sql
+\d cagg_rides_view
+                          View "public.cagg_rides_view"
+      Column       |            Type             | Collation | Nullable | Default 
+-------------------+-----------------------------+-----------+----------+---------
+ vendor_id         | text                        |           |          | 
+ day               | timestamp without time zone |           |          | 
+ total_rides       | bigint                      |           |          | 
+ avg_fare          | numeric                     |           |          | 
+ max_trip_distance | numeric                     |           |          | 
+ min_trip_distance | numeric                     |           |          | 
+```
+
+We can view the metadata for the continuous aggregate in the 
+**timescaledb_information.continuous_aggregates** view.
+
+``` sql
+SELECT view_name, refresh_lag, refresh_interval, max_interval_per_job, materialization_hypertable 
+FROM timescaledb_information.continuous_aggregates;
+-[ RECORD 1 ]--------------+-------------------------------------------------
+view_name                  | cagg_rides_view
+refresh_lag                | 02:00:00
+refresh_interval           | 00:30:00
+max_interval_per_job       | 20:00:00
+materialization_hypertable | _timescaledb_internal._materialized_hypertable_2
+```
+The refresh_interval is set to 30 minutes. The computed aggregates are saved 
+in the materialization table,  _timescaledb_internal._materialized_hypertable_2. 
+
+What is refresh_lag and max_interval_per_job? We use the 
+`timescaledb.refresh_lag` parameter to indicate by how much does the continuous
+ aggregate query lag behind the data in the *rides* table. For example, if we 
+expect frequent updates to the *rides*  table for the current hour, we do not 
+want to precompute the aggregates for that range. We would set the 
+**refresh_lag = ‘1h’** to indicate that. (The default value is twice the 
+bucket_width used by the time_bucket expression. This is the 2 hours shown for 
+refresh_lag in the view output above). So the continuous aggregate will get 
+refreshed every 30 minutes (refresh_interval) but will update the continuous 
+aggregates only for the data that satisfies the condition: 
+time_bucket(‘1h’, pickup_datetime) <   max(pickup_time) - ‘1h’  (if the 
+refresh_lag is set to 1 hour)
+
+The `timescaledb.max_interval_per_job` parameter is used when we want to limit 
+the amount of data processed by an update of the continuous aggregate and use 
+smaller or bigger batch sizes (the batching is done automatically by TimescaleDB). 
+`timescaledb.`max_interval_per_job` specifies the batch size.
+
+refresh_lag and max_interval_per_job are additional parameters that can be 
+specified while creating or altering a continuous aggregate. Refer to the 
+documentation for the syntax.
```

Comment:
Convert "documentation" to a link?

Reply:
Are you leaving it as text b/c that section of the docs hasn't been written yet?